web-interface/api/api-info.json

```diff
@@ -83,6 +83,10 @@
         "Description": "Kill all docker-containers",
         "Example":"curl rpms:5000/api/DoKillDocker -X post"
     },        
+    {   "EndPoint": "POST api/DoStartDocker",
+        "Description": "(Re)Start all docker-containers",
```

Comment:
The command StartDocker does not actually restart a running container. Running containers are left alone, Only stopped (paused) container will be started. 

So a better description would be: Start all (stopped) containers